docs/storage_v2/built_in_features/supercontract/sc_files.md

```diff
@@ -0,0 +1,11 @@
+---
+id: sc_files
+title: SC files
+---
+
+There are two file types:
```

Comment:
Do we still use these file types (especially wat) and there repo https://github.com/WebAssembly/wabt/tree/1.0.13?